erc190/package.py

```diff
@@ -29,6 +29,23 @@ def __init__(self, lockfile):
 
         try:
             validate(package_data, schema_data)
-        except jsonValidationError:
+        except:
             raise ValidationError
         self.package_data = package_data
+
+    def __repr__(self):
+        """
+        This can be expanded to include the
+        hashed content as discussed once
+        Package.hash exists
+        """
+
+        name = self.package_data['package_name']
```

Comment:
Can you change this and the `version` lookup to use the properties (`self.name` and `self.version`)